opencti-platform/opencti-front/src/components/ItemAccountStatus.tsx

```diff
@@ -72,6 +76,11 @@ const ItemAccountStatus: FunctionComponent<ItemAccountStatusProps> = ({
   account_status,
   variant,
 }) => {
+  const { me: { monochrome_labels } } = useAuth();
+  const { palette: { mode } } = useTheme();
+  const theme = mode === 'dark'
```

Comment:
I don't understand your code. 
```const theme = useTheme<Theme>();```
should be enough

Reply:
In that case I think you just missed the <Theme> from components/Theme.d.ts in the useTheme that handles typescript correctly :)